charts/kyverno/crds/crds.yaml

```diff
@@ -3,6 +3,16 @@ kind: CustomResourceDefinition
 metadata:
   name: clusterpolicies.kyverno.io
 spec:
+  additionalPrinterColumns:
+  - JSONPath: .spec.background
+    description: Background provides choice for applying rules to existing resources.
```

Comment:
update description to "Background controls if rules are applied to existing resources during a background scan."